src/default-log-formats.json

```diff
@@ -68,6 +68,37 @@
             }
         ]
     },
+    "rbd_log" : {
+        "title" : "VMware vSphere Auto Deploy log format",
+        "regex" : {
+            "std" : {
+                "pattern" : "^(?<timestamp>\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}(?:\\.\\d{3})?) \\[(?<pid>\\d+)\\](?<level>\\w+):(?<module>\\w+):(?<body>.*$)"
+            }
+        },
+        "level-field" : "level",
+        "timestamp-field" : "timestamp",
+        "level" : {
+            "error" : "ERROR",
+            "debug" : "DEBUG",
+            "info" : "INFO",
+            "warning" : "WARNING"
+        },
+        "value" : {
+            "pid" : {
+                "kind" : "integer",
+                "identifier" : true
+            },
+            "module" : {
+                "kind" : "string",
+                "identifier" : true
+            }
+        },
+        "sample" : [
+            {
+                "line" : "2015-04-24T21:09:29.296 [25376]INFO:somemodule:Something very INFOrmative."
```

Comment:
I believe generic_log format for Auto Deploy broke with the check-in (ae64b599bd8342a077722d04659a1b4081e18420). I don't really understand the code well enough, but in that change you added the start-of-line(^) and end-of-line($) anchors to the LEVEL_RE for some reason. As far as I can tell from putting in log statements, that RE is matched against the complete line and the level is in the middle of the line, so the string2level just returns LEVEL_UNKNOWN for every line. Was there a reason why you added those anchors?